src/AppBundle/Resources/config/services.yml

```diff
@@ -21,10 +21,11 @@ services:
 
     prooph.api_command_controller:
         class: Prooph\AppBundle\Controller\ApiCommandController
-        arguments: ['@prooph_service_bus.command_bus.main_bus', '@prooph.message_factory']
+        arguments: ['@prooph_service_bus.command_bus.todo_bus', '@Prooph\Common\Messaging\MessageFactory']
```

Comment:
same as above, we should change the service id of the message factory to a lower case dot separated name